extensions/mentions/extend.php

```diff
@@ -80,13 +87,14 @@
         ->load([
             'mentionsUsers', 'mentionsPosts', 'mentionsPosts.user', 'mentionedBy',
             'mentionedBy.mentionsPosts', 'mentionedBy.mentionsPosts.user', 'mentionedBy.mentionsUsers',
+            'mentionsGroups'
         ]),
 
     (new Extend\ApiController(Controller\CreatePostController::class))
-        ->addInclude(['mentionsPosts', 'mentionsPosts.mentionedBy']),
+        ->addInclude(['mentionsPosts', 'mentionsPosts.mentionedBy', 'mentionsGroups']),
```

Comment:
I believe since groups are preloaded on the frontend we don't need them to be included in responses by default, s an optional include fits better.
```suggestion
        ->addInclude(['mentionsPosts', 'mentionsPosts.mentionedBy'])
        ->addOptionalInclude('mentionsGroups'),
```